packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -27,7 +28,7 @@ const Options = styled.div<{ isInLine: boolean }>`
     display: flex;
     flex: ${({ isInLine }) => isInLine && '1'};
     padding: 0 4px;
```

Comment:
Is there some rule where we use this values from `spacingsPx` and where we just put them like this. Saw multiple places where the value for padding, margin, text-spacing etc. can be used from `spacingsPx`.

Reply:
`SelectBar` is updated in another PR :) In general, I would always use the `spacings` tokens, unless the designs use custom spacings. Which is rarely the case with the new designs.

Reply:
Ok, just added last stuff regarding this